go/noise/gaussian_noise.go

```diff
@@ -143,6 +161,14 @@ func checkArgsGaussian(label string, l0Sensitivity int64, lInfSensitivity, epsil
 	return checks.CheckDeltaStrict(label, delta)
 }
 
+// checkArgsConfidenceIntervalGaussian checks the parameters for gaussian confidence interval, as well as the provided confidence level.
+func checkArgsConfidenceIntervalGaussian(label string, l0Sensitivity int64, lInfSensitivity, epsilon, delta, alpha float64) error {
+	if err := checks.CheckAlpha(label, alpha); err != nil {
+		return err
+	}
+	return checkArgsGaussian(label, l0Sensitivity, lInfSensitivity, epsilon, delta)
+}
+
```

Comment:
I believe this should live in checks.go

Reply:
I also have the same comment Karim mentioned, so i will wait for your response in his PR :) Thank you!